### PR TITLE
Gem implementation

### PR DESCRIPTION
**Pull request by akilmadan**

### CODE REVIEW

**Comment by msmirnoff** [comment]
The first commit is not really for reviewing as that is in its own separate PR... so why is it even here? Given that the 3 other commits are adding new files, the original commit does not need to be here. And it can cause merge conflicts if you change the boilerplate before merging this PR.

**Comment by msmirnoff** [comment]
Implementation LGTM, :+1:

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+$LOAD_PATH.unshift File.expand_path('../lib', __FILE__)
+
+MINIMUM_COVERAGE = 80
+
+unless ENV['COVERAGE'] == 'off'
```
Comment (mnoack) [inline spec/spec_helper.rb]: Can we move coverage to support/coverage_loader.rb as in all other gems now (I just did this yesterday for 10-20 gems)

Also see how they do SimpleCov.formatter for the nicer syntax.

Comment (akilmadan) [inline spec/spec_helper.rb]: Done!

**Comment by akilmadan** [comment]
Hey @msmirnoff  the first commit showed up as this was branched off that work. I should've made a note on the dependency of the PRs. The PR looks a lot cleaner now that we've merged the original branch in.

**Comment by msmirnoff** [comment]
@vgmaster21 
The PR: :+1: 
But I don't understand how this PR was dependent on the other PR :smile:

**Comment by alxberardi** [comment]
:+1:

**Pull request merged**